src/mgos_imu_mpu6886.c

```diff
@@ -1,31 +1,20 @@
-/*
```

Comment:
I see that you've now entirely removed the copyright and its license, this is also not intended and will confuse readers. The file should reflect who wrote it and whose copyright it is, I would surmise that is you, so this would read """Copyright 2020 Sergio R. Caprile <scaprile@gmail.com>""" and the Apache 2.0 license header text.

Reply:
done